__tests__/IncidentCard.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+
+import IncidentCard from '../src/Components/IncidentList/IncidentCard.Component';
+import IncidentList from '../src/Components/IncidentList/IncidentList.Component';
+
+
```

Comment:
Why r we still testing with snapshots?

Reply:
@Mbarak-Mbigo incidents snapshot tests removed.